storage/innobase/xtrabackup/test/t/xbcloud.sh

```diff
@@ -122,9 +122,17 @@ fi
 #PXB-2198 xbcloud doesn't return the error on delete if the backup doesn't exist in s3 bucket
 xbcloud --defaults-file=$topdir/xbcloud.cnf delete somedummyjunkbackup 2>$topdir/pxb-2198.log
 
-if ! grep -q failed $topdir/pxb-2198.log ; then
+if ! grep -q "error: backup" $topdir/pxb-2198.log ; then
     die 'xbcloud did not exit with error on delete'
 fi
+
+#PXB-2202 Xbcloud does not display an error when xtrabackup fails to create a backup
+true | xbcloud --defaults-file=$topdir/xbcloud.cnf put somedummyjunkbackup 2>$topdir/pxb-2202.log
```

Comment:
why this "true | " ?

Reply:
It could be any command which is successful but doesn't produce any files to upload backup to the cloud. 
previously we would get message that upload is successful.

Reply:
I would also prefer a command with xtrabackup that fails

Reply:
implemented